### PR TITLE
Squashed commits into one update

### PR DESCRIPTION
**Pull request by gabn88**
Update jQuery.colorPicker.js

Update tests.py

### CODE REVIEW

**Comment by charettes** [comment]
The changes have been merged in b617ad29356cea4505f32734c88fc654f7d605ab thanks for your contribution.

You should really learn how to rebase a branch and squash commits since opening a new PR in every iteration of your patch makes it hard to follow.

**Comment by gabn88** [comment]
Yes, I know and I'm sorry. I tried to rebase and squash, and I thought I succeeded, but now I still created a new PR... Do you know why that happened? Do you maybe know a good tutorial?

Thank you!

**Comment by charettes** [comment]
Don't worry I also struggled to get my head around it at first.

I don't remember the exact tutorial I followed but (this one)[https://wincent.com/wiki/Git_rebase_explained] looks promising.

**Comment by gabn88** [comment]
Just one final question: If I now download the django-colorful package with pip, it still does not include the changes. Did I do something wrong? Or does pushing to pypi not go automatically from github?

**Comment by charettes** [comment]
@gabn88, I just uploaded a new version of `django-colorful` package which contains your changes.
